.github/workflows/tests.yml

```
@@ -37,17 +37,33 @@ jobs:
       - name: Install Bundler RubyGems
         run: brew install-bundler-gems --groups=sorbet
 
-      - name: Install shellcheck
-        run: brew install shellcheck
+      - name: Install linting formulae
+        run: brew install shellcheck shfmt vale
 
       - run: brew style --display-cop-names
 
       - run: brew typecheck
 
       - name: Run vale for docs linting
+        run: vale docs/
+
+      - name: Install Ruby for linting tools
+        uses: ruby/setup-ruby@v1
```

```suggestion
        uses: ruby/setup-ruby@v2
```

or even `@main` or `@master`

---

Er do I'm remember the old actions one I think. Ignore me 😓